Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -267,11 +273,23 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
 void MenuBar::UpdateToolsMenu(bool emulation_started)
 {
   const bool enable_wii_tools = !emulation_started || !SConfig::GetInstance().bWii;
```

Comment:
I'm sorry, but could you remove this and just disable Wii tools whenever emulation is running (even when it's a GC game)? I just realised that running a GC game in BC mode will cause this to be true, even though the NAND is still initialised and used. (I made a mistake here, but note that this PR will cause an assertion to fail)